### PR TITLE
chore(docs): set stability badge to "stable"

### PR DESCRIPTION
**Pull request by ryparker**
## Summary

This PR removes the use of any beta versions and updates the stability badge on the repo's README to "Stable"

### CODE REVIEW

File: docs/plus/index.md [hunk 1]
```diff
@@ -0,0 +1,338 @@
+# cdk8s+ v25
```
Comment (iliapolo) [inline docs/plus/index.md]: Why did you add this file here? It exists in the website repo

**Review by iliapolo — CHANGES_REQUESTED** [review]

File: docs/plus/index.md (hunk 1 above)
Comment (ryparker) [reply docs/plus/index.md]: Just removed it. Forgot we moved this back to cdk8s.

**Review by iliapolo — APPROVED** [review]

**Comment by iliapolo** [comment]
@ryparker add the back port labels?
